elizaOS/eliza
# docs: streamline model support description in README

# Relates to:

N/A (This is a standalone update to improve the `README.md` file. No issue or ticket is directly linked.)



# Risks

Low. This change only updates documentation and does not affect code or functionality.

# Background

## What does this PR do?

This PR updates the `README.md` file to remove redundancy in the description of supported models and streamline the list of features for clarity.

## What kind of change is this?

Documentation update (non-breaking change that improves documentation).



# Documentation changes needed?

My changes do not require a change to the project documentation.



# Testing

## Where should a reviewer start?

Start by reviewing the `README.md` file changes to ensure clarity and correctness in the new wording.

## Detailed testing steps

1. Open the updated `README.md` file.
2. Review the list of features for accuracy and readability.
3. Verify that no redundant information is present.
